src/api/wallet/WalletApiImpl.ts

```diff
@@ -134,7 +134,7 @@ const closeOpenWebSocketConnection = (web3: Web3): void => {
 export class WalletApiImpl implements WalletApi {
   private _listeners: ((walletInfo: WalletInfo) => void)[]
   private _provider: Provider | null
-  private _web3: Web3
+  public _web3: Web3
```

Comment:
Why `public`?

Reply:
I am returning it inside the `useWalletInfo` hook. Thoughts?

Reply:
That shouldn't affect anything
Public is for `new WalletApiImpl().web3`
Returns are unrelated

Reply:
:+1: